src/main/scala/org/allenai/deep_qa/experiments/datasets/Science.scala

```diff
@@ -83,6 +136,51 @@ object ScienceDatasets {
     ScienceCorpora.buscElasticSearchIndex(3)
   )
 
+  /*
+   * Fourth Grade Questions in RC Format (BUSC-fetched passage,
+   * question, answer options)
+   */
+  val readingComprehensionOmnibusQaGradeFourTrainQuestionsWithBuscBackground: JValue =
+    ("sentence producer type" -> "qa and background to mc") ~
```

Comment:
I'm just noticing that you named this type "qa and background to mc".  It should be "qa and background to rc".